integration-tests/tests_aliens.py

```diff
@@ -0,0 +1,97 @@
+parser.add_argument('-c', dest='count', type=int, help='amount of entries to process', required=True)
+parser.add_argument('-l', dest='payload', type=int, help='payload in bytes', required=True)
+parser.add_argument('-n', dest='node', type=str, help='target node address', required=True)
+parser.add_argument('-f', dest='first', type=int, help='first index', default=0)
+parser.add_argument('-t', dest='threads', type=int, help='amount of working threads', default=1)
+parser.add_argument('--mode', dest='mode', type=str, help='random or normal', choices=['random', 'normal'], default='normal')
+parser.add_argument('-k', dest='keysize', type=int, help='size of binary key (8 or 16)', choices=[8, 16], default=8)
+
+parsed_args = parser.parse_args()
+
+#get container object mapping to ports
+container_dict = {}
+try:
+    for i in range(int(bob_nodes_amount_string)):
+        port_num = str(20000+i)
+        container_dict[port_num] = str(d_cli.container.list(filters={'publish':f'{port_num}'})[0].id)
+except KeyError:
+    sys.exit('Nodes amount is not set.')
+except ValueError:
+    sys.exit('Amount of nodes has unexpected value.')
+
+#runs put and stops nodes in cycle
+written_count = 0
+try:
+    upper_bound = int(bob_nodes_amount_string)
+    for i in range(1, upper_bound):
+        #make correctly formatted args 
+        dict_args = make_run_args(parsed_args, written_count)
+        bobp_args = args_to_str(dict_args)
+        #run put
+        print(f'Running bobp -b put {bobp_args.rstrip()}')
+        p = subprocess.check_output(shlex.split(f'./bobp -b put {bobp_args.rstrip()}')).decode('ascii')
+        print(str(p))
+        if f'put errors:' in str(p) or f'panicked' in str(p):
+            sys.exit(f'Put test failed, see output.')
+        written_count += dict_args.get('-c')
+        #stops one
+        sleep(30)
+        d_cli.container.stop(container_dict[str(20000 + i)])
+        print(f'Bob node {i} stopped.\n')
+        stopped_list = d_cli.container.list(filters={"status":"exited"})
+        print('Stopped containers:\n')
+        for i in range(len(stopped_list)):
+            print(f'{stopped_list[i].id}\n') 
+except subprocess.CalledProcessError as e:
+    sys.exit(str(e.stderr))
+
+try:
+    for key, value in container_dict.items():
+        print(f'Starting node on port {key}...')
+        d_cli.container.start(value)
+except DockerException as e:
+    sys.exit(e.stderr)
+
+try:
+    ensure_backend_up(int(bob_nodes_amount_string))
+except ValueError:
+    sys.exit('Amount of nodes has unexpected value.')
+
+try:
+    dict_args = make_run_args(parsed_args, 0)
+    dict_args['-c'] = str(written_count)
+    bobp_args = args_to_str(dict_args)
+    print(f'Running bobp -b get {bobp_args.rstrip()}')
+    p = subprocess.check_output(shlex.split(f'./bobp -b get {bobp_args.rstrip()}')).decode('ascii')
+    print(str(p))
+    if f'get errors:' in str(p) or f'panicked' in str(p):
```

Comment:
It is better to check for success, not for errors. Because if the output of `bobp` someday will change, the test start passing by even on errors. I suggest to search for `total err: 0`